pulseapi/creators/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
It was generated automatically with `manage.py startapp`, I'm fine leaving this as a stub for future tests